lxd/images.go

```diff
@@ -3652,36 +3664,22 @@ func imageSyncBetweenNodes(d *Daemon, project string, fingerprint string) error
 	// Select the right project
 	client = client.UseProject(project)
 
-	createArgs := &lxd.ImageCreateArgs{}
-	imageMetaPath := shared.VarPath("images", fingerprint)
-	imageRootfsPath := shared.VarPath("images", fingerprint+".rootfs")
-
-	metaFile, err := os.Open(imageMetaPath)
+	_, image, err := d.cluster.GetImage(project, fingerprint, false)
 	if err != nil {
-		return err
+		return errors.Wrap(err, "Failed to get image")
 	}
-	defer metaFile.Close()
-
-	createArgs.MetaFile = metaFile
-	createArgs.MetaName = filepath.Base(imageMetaPath)
-
-	if shared.PathExists(imageRootfsPath) {
-		rootfsFile, err := os.Open(imageRootfsPath)
-		if err != nil {
-			return err
-		}
-		defer rootfsFile.Close()
 
-		createArgs.RootfsFile = rootfsFile
-		createArgs.RootfsName = filepath.Base(imageRootfsPath)
+	args := lxd.ImageCopyArgs{
+		AutoUpdate:  image.AutoUpdate,
+		CopyAliases: true,
```

Comment:
I doubt we need this as we're only copying the raw image, so don't care too much about aliases.

Reply:
For that matter, none of this struct should be needed to be passed except for `Mode` and `Type`.
The DB entry creation phase will be skipped on the receiver so no need for that extra stuff.